notebooks/config.ipynb

```diff
@@ -0,0 +1,1046 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "d023d283-7e1d-4e75-95b8-5033bea42a59",
+   "metadata": {},
+   "source": [
+    "# Config Tool\n",
+    "\n",
+    "The `uwtools` API's `config` module provides functions to create and manipulate configuration files, objects, and dictionaries.\n",
+    "\n",
+    "For more information, please see the <a href=\"https://uwtools.readthedocs.io/en/main/sections/user_guide/api/config.html\">uwtools.api.config</a> Read the Docs page.\n",
+    "\n",
+    "## Table of Contents\n",
+    "\n",
+    "* [Getting Config Objects](#Getting-Config-Objects)\n",
+    "* [Config Depth Limitations](#Config-Depth-Limitations)\n",
+    "* [Realizing Configs](#Realizing-Configs)\n",
+    "    * [Updating configs](#Updating-configs)\n",
+    "    * [Using the `key_path` parameter](#Using-the-key_path-parameter)\n",
+    "    * [Using the `values_needed` parameter](#Using-the-values_needed-parameter)\n",
+    "    * [Using the `total` parameter](#Using-the-total-parameter)\n",
+    "* [Realizing Configs to a Dictionary](#Realizing-Configs-to-a-Dictionary)\n",
+    "<!--cell 0-->"
```

Comment:
Adding notebook cells that intentionally raise an exception made it necessary to target specific cell indices with testbook, so I added a comment to each markdown cell with its index.